### PR TITLE
Feature/saf 3478/password input

### PR DESCRIPTION
**Pull request by daiana-simplenight**
Add password input component and eye icon

### CODE REVIEW

File: src/components/Inputs/PasswordInput/PasswordInput.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        onChange={onChange}
+        rightIcon
+      />
+      <button
+        className="absolute w-5 h-10 right-4"
+        onClick={handleShow}
+        type="button"
+      >
+        <Eye className="text-dark-1000" />
+      </button>
```
Comment (isabel-light-it) [inline src/components/Inputs/PasswordInput/PasswordInput.tsx:40]: could the size of the icon depend on the size of the input. I believe small inputs use 16x16 and large use 24x24

Comment (daiana-simplenight) [reply src/components/Inputs/PasswordInput/PasswordInput.tsx:40]: That's true! I will fix it

**Review by isabel-light-it — APPROVED** [review]

File: src/components/Inputs/BaseInput/BaseInput.tsx [hunk 2]
```diff
@@ -19,9 +24,11 @@ const BaseInput = ({
   state = 'idle',
   value,
   onChange,
+  rightIcon = false,
 }: BaseInputProps) => {
   const height = size === 'small' ? 'h-8' : 'h-11';
```
Comment (NakaSimpleNight) [inline src/components/Inputs/BaseInput/BaseInput.tsx:29]: Why are we using height?

**Review by NakaSimpleNight — CHANGES_REQUESTED** [review]

File: src/components/Inputs/BaseInput/BaseInput.tsx (hunk 2 above)
Comment (daiana-simplenight) [reply src/components/Inputs/BaseInput/BaseInput.tsx:29]: Because we have large and small size

**Pull request merged**